extensions/sql-migration/src/dashboard/sqlServerDashboard.ts

```diff
@@ -0,0 +1,510 @@
+
+				const tasksContainer = await this.createTasks(view);
+
+				container.addItem(header, {
+					CSSStyles: {
+						'background-image': `url(${vscode.Uri.file(this.asAbsolutePath('images/background.svg'))})`,
+						'width': `1100px`,
+						'height': '300px',
+						'background-size': '100%',
+					}
+				});
+				header.addItem(tasksContainer, {
+					CSSStyles: {
+						'width': `${maxWidth}px`,
+						'height': '150px',
+					}
+				});
+
+				header.addItem(await this.createFooter(view), {
+					CSSStyles: {
+						'margin-top': '20px'
+					}
+				});
+
+				const mainContainer = view.modelBuilder.flexContainer()
+					.withLayout({
+						flexFlow: 'column',
+						width: '100%',
+						height: '100%',
+						position: 'absolute'
+					}).component();
+				mainContainer.addItem(container, {
+					CSSStyles: { 'padding-top': '25px', 'padding-left': '5px' }
+				});
+				await view.initializeModel(mainContainer);
+
+				this.refreshMigrations();
+				resolve();
+			});
+		});
+	}
+
+	private createHeader(view: azdata.ModelView): azdata.FlexContainer {
+		const header = view.modelBuilder.flexContainer().withLayout({
+			flexFlow: 'column',
+			width: maxWidth,
+		}).component();
+		const titleComponent = view.modelBuilder.text().withProperties({
+			value: loc.DASHBOARD_TITLE,
+			CSSStyles: {
+				'font-size': '36px',
+				'font-family': 'Segoe UI'
+			}
+		}).component();
+		const descComponent = view.modelBuilder.text().withProperties({
+			value: loc.DASHBOARD_DESCRIPTION,
+			CSSStyles: {
+				'font-size': '12px',
+			}
+		}).component();
+		header.addItems([titleComponent, descComponent], {
+			CSSStyles: {
+				'width': `${maxWidth}px`,
+				'padding-left': '20px'
+			}
+		});
+
+		return header;
+	}
+
+	private asAbsolutePath(filePath: string): string {
+		return path.join(this._root || '', filePath);
+	}
+
+	private async createTasks(view: azdata.ModelView): Promise<azdata.Component> {
+		const tasksContainer = view.modelBuilder.flexContainer().withLayout({
+			flexFlow: 'row',
+			width: '100%',
+			height: '50px',
+		}).component();
+
+		const migrateButtonMetadata: IActionMetadata = {
+			title: loc.DASHBOARD_MIGRATE_TASK_BUTTON_TITLE,
+			description: loc.DASHBOARD_MIGRATE_TASK_BUTTON_DESCRIPTION,
+			iconPath: {
+				dark: this.asAbsolutePath('images/migration.svg'),
+				light: this.asAbsolutePath('images/migration.svg'),
+			},
+			command: 'sqlmigration.start'
+		};
+		const migrateButton = this.createTaskButton(view, migrateButtonMetadata);
+
+		const preRequisiteListHTMLTemplate =
+			`
+			<span style="font-size: 14px;">Things you need before starting migration:</span><br>
+			<ul style="padding:0 0 0 10px;font-size:12px;">
+			<li>Azure account details</li>
+			<li>Azure SQL Managed Instance or SQL Server on Azure Virtual Machine</li>
+			<li>Backup location details</li>
+			</ul>
+			`;
+
+		const preRequisiteListElement = view.modelBuilder.dom().withProps({
+			html: preRequisiteListHTMLTemplate
```

Comment:
@alanrenmsft Do you know why we have this? This is dangerous for extensions to have direct access to...

@aasimkhan30 What can't you use direct modelview components for with this?

Reply:
this component is introduced by this PR: https://github.com/microsoft/azuredatastudio/pull/2622, Abbie didn't give description about why is this component needed, Matt did raise the same concern as you did in that PR. 

this issue is mentioned in this PR, so I guess it is for markdown related stuff? https://github.com/microsoft/azuredatastudio/issues/2626

@chlafreniere do you remember?

Reply:
Pinged some details offline; this component was used in markdown rendering for a previous notebook implementation that is no longer needed any longer; we currently do markdown rendering in-proc.

Reply:
I have switched to text component now. 

Now:
<img width="902" alt="image" src="https://user-images.githubusercontent.com/6816294/108167352-67040500-70aa-11eb-975f-23ae82193e8c.png">

Before:
<img width="889" alt="image" src="https://user-images.githubusercontent.com/6816294/108167506-a6325600-70aa-11eb-9b5c-61babcbfcbfa.png">